config/src/main/java/com/yahoo/config/subscription/impl/JRTConfigRequester.java

```diff
@@ -53,8 +53,9 @@
     private final ConnectionPool connectionPool;
     private final ConfigSourceSet configSourceSet;
 
-    private Instant noApplicationWarningLogged = Instant.MIN;
+    private Instant timeForLastLogWarning;
     private int failures = 0;
+    private boolean closed = false;
```

Comment:
Should this be atomic or volatile? It's only used for logging and testing now, but maybe more later?

Reply:
Yes, good point, will fix